### PR TITLE
Chunked glitch loader

### PR DESCRIPTION
**Pull request by EthanMarx**
Adds a chunked glitch loader

### CODE REVIEW

File: projects/sandbox/train/train/train.py [hunk 1]
```diff
@@ -322,6 +326,25 @@ def main(
...
+
+    # find glitch paths and initiate glitch loader
+    glitch_paths = {}
+    for ifo in ifos:
+        ifo_dir = glitch_dir / ifo
+        glitch_paths[ifo] = list(ifo_dir.iterdir())
+
+    glitch_loader = ChunkedGlitchDataset(
+        reads_per_chunk=10,
+        batches_per_chunk=int(batches_per_epoch / 4),
```
Comment (wbenoit26) [inline projects/sandbox/train/train/train.py:339]: Does the denominator need to match `chunks_per_epoch` below?

Comment (EthanMarx) [reply projects/sandbox/train/train/train.py:339]: Yeah I think `batches_per_chunk * chunks_per_epoch` (i.e. batches_per_epoch) should be the same for both the `background_loader` and `glitch_loader`

File: projects/sandbox/train/train/train.py [hunk 2]
```diff
@@ -44,6 +46,8 @@ def main(
     highpass: float,
     # augmentation args
     waveform_prob: float,
+    glitch_prob: float,
```
Comment (wbenoit26) [inline projects/sandbox/train/train/train.py:49]: For consistency, I think we should rename this to `glitch_frac`, as we now have a guaranteed number of glitches per batch

**Comment by wbenoit26** [comment]
No issues with the functionality that I can see. I am going to request that you update the docstrings in `background.py` and `glitches.py` to reflect any new/removed arguments and any changes to returned values.

**Comment by wbenoit26** [comment]
Nice, looks good. What are your thoughts on tests? I know we need to redo the training tests, so do you want to add tests for this stuff when that happens?

**Comment by EthanMarx** [comment]
Yeah I think adding tests to the train project will happen when we migrate the `ml4gw` `Whitener` and `ChunkedDataset`. So maybe this should wait til then, rebase, and add tests for all of these features together.

**Comment by EthanMarx** [comment]
closing in favor of #462

**Pull request closed without merging**